scripts/githooks/secret-scan-pre-commit.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -e
+
+# Pre-commit git hook to scan for secrets hardcoded in the codebase.
+#
+# Usage:
+#   $ ./secret-scan-pre-commit.sh
+#
+# Options:
+#   ALL_FILES=true  # Scan whole git history or 'last-commit', default is `false`
+#   VERBOSE=true    # Show all the executed commands, default is `false`
```

Comment:
Isn't verbose always true since we pass it as a flag to the cmd - i.e. cmd="protect --source=/scan --verbose --staged"